ext/build_defaults.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-packaging_url: 'git://github.com/puppetlabs/packaging.git --branch=1.0.x'
+packaging_url: 'git://github.com/puppetlabs/packaging.git --branch=master'
```

Comment:
We decided that since we're planning to merge the 1.0.x branch back into master after thorough testing, this was unnecessary and would just create more churn in the long run. The puppet-agent repo will be switched to the 1.0.x branch, and will be switched back to master once the merge happens.